content/user/admin_pages/localization/languages.md

```diff
@@ -5,10 +5,13 @@ weight: 20
 ---
 
 This page allows you to add additional language(s) in order to internationalize the output generated by Zen Cart.
+
 ## New Language
 
 To add a new language, click on the _new language_ button. A form will appear. Enter the information, then click on the _insert_ button to add the new language. Click on the _cancel_ button if you change your mind.
 
+Adding the new language also populates the `orders_status` table with order statuses in the *current* language.  Go to Admin > Localization > Orders Status and update them to the new language. 
```

Comment:
And POSM too, if it's installed. `Localization` >`Out-of-Stock Labels`

Reply:
Thanks - updated.